.env.sample

```diff
@@ -1,8 +1,10 @@
-NEXT_PUBLIC_BITCOIND_URL=
-NEXT_PUBLIC_EMILY_URL=
-NEXT_PUBLIC_MEMPOOL_API_URL=
-NEXT_PUBLIC_BITCOIN_RPC_USER_NAME=
-NEXT_PUBLIC_BITCOIN_RPC_PASSWORD=
-NEXT_PUBLIC_WALLET_NETWORK=
-NEXT_PUBLIC_SIGNER_AGGREGATE_KEY=
-NEXT_PUBLIC_CONTRACTS_DEPLOYER=
+BITCOIND_URL=
+EMILY_URL=
+MEMPOOL_API_URL=
+BITCOIN_RPC_USER_NAME=
+BITCOIN_RPC_PASSWORD=
+WALLET_NETWORK=
+SBTC_CONTRACT_ADDRESS=
+SBTC_CONTRACT_NAME=
```

Comment:
I left a comment about these in another PR but I don't know if it was addressed.

You should only need to contract deployer. Then, the contract name is constant and you can get the full principal address.